timesheets/overview/get_started.rst

```diff
@@ -0,0 +1,66 @@
+===========================
+Get Started with Timesheets
+===========================
+
+Filling up timesheets allows you to accurately track the time spent on tasks making sure your
+project is profitable. It also allows you to make fair comparisons with forecasted times and to
+become more proactive, as you can do it it from any device, anywhere.
+
+Choose an encoding unit
+=======================
+
+| Under :menuselection:`Timesheets --> Configuration --> Settings`, choose the unit of measure to
+  register your timesheet. This provides tools and widgets to help you encoding.
+| *Minimal duration* is the minimum time at which a task is recorded.
+| *Rounding up* allows you to time up or down to the nearest minutes or hours. For example: if the
+  interval is set to 30min, a 14min entry will show up in reports as 0min, and a 29min entry
+  as 30min.
+
+.. image:: media/get1.png
+   :align: center
+   :alt: Set the time unit used to record your timesheets in Odoo Timesheets application
+
+.. note::
+   Reportings are expressed in hours, the default value.
+
+Create a task from a SO & timesheet on it
+=========================================
+
+| Under the *General Settings* of your product, categorize it as a service.
+| On the *Sales* tab, (section *Sales Invoicing Policy*) choose if you want to invoice customers
+  based on the quantity previous ordered (the one sent on a quotation, for instance), on the time
+  recorded (while the service was being executed), or by manually adding the time spent when
+  creating the invoice.
+| Chose the right *Service Tracking* for you and, from now on, when a sales order is confirmed,
+  Odoo automatically creates a task/project (if that was the chosen option) and allows you to
+  timesheet on it.
```

Comment:
```suggestion
| Under the *General Settings* of your product, categorize it as a service.
| On the *Sales* tab, (section *Sales Invoicing Policy*) choose if you want to invoice customers
  based on the quantity previously ordered (the one sent on a quotation, for instance), on the time recorded (while the service was being executed), or by manually adding the time spent when creating the invoice.
| Choose the right *Service Tracking* for you and, from now on, when a sales order is confirmed,
  Odoo automatically creates a task/project (if that was the chosen option) and allows you to timesheet on it.
```

Reply:
I think I take parts that are too big... so it doesn't show what are the changes I suggest...